geopolars/src/geoseries.rs

```diff
@@ -162,6 +162,13 @@ pub trait GeoSeries {
     /// ```
     fn skew(&self, xs: f64, ys: f64, origin: TransformOrigin) -> Result<Series>;
 
+    /// Returns a Series containing the distance to aligned other. The operation works on a 1-to-1 row-wise manner.
```

Comment:
It would be great for the docstring for here and for the Python API to make clear that it's calculating cartesian distance in 2D space, and the units of the output are in terms of the CRS of the two input series. (I'm surprised the geopandas docstring doesn't mention that)